packages/web/pages/guilds.tsx

```diff
@@ -22,10 +23,44 @@ const GuildsPage: React.FC<Props> = ({ guilds }) => (
   <PageContainer>
     <HeadComponent
       title="MetaGame Guilds"
-      description="MetaGame is a Massive Online Coordination Game! Guilds participating in MetaGame…."
+      description="MetaGame is a Massive Online Coordination Game! Guilds participating in MetaGame…"
       url="https://my.metagame.wtf/guilds"
     />
-    <GuildList {...{ guilds }} />
+
+    {/* VStack is used to make a consistent gap between the Join CTA and the Guilds list */}
+    <VStack maxW="7xl" w="100%" spacing={{ base: 6, md: 8, lg: 12 }} pb={8}>
```

Comment:
maybe the spacing is increasing unnecessarily and we could keep it at 6. i've taken a look at other pages and it seems inconsistent and pretty big in some cases.
for example: https://frontend-pr-1484-mjhnbmqqna-uk.a.run.app/roles
or: https://frontend-pr-1484-mjhnbmqqna-uk.a.run.app/raids

Reply:
but we could do this in another PR after talking to the designers

Reply:
@davort @peth-yursick what do you think about this?

Reply:
You're talking about this part, yeah?

![image](https://user-images.githubusercontent.com/92962228/209962079-c48999fc-c003-4e44-b4a0-25c015187ede.png)

After looking at the pages you linked, yes I agree the spacing could be tighter.

Dave had mentioned reworking the design around the search box as a separate task
-> https://app.dework.xyz/metagame-1/metafest-2022?taskId=7e06aaa7-6f16-4cfd-a231-8d0882829ef4

Doing that as a separate task would let us get the join links in quickly, which was the main point of the issue originally :D

Reply:
But I can make it match the roles and raids pages more closely as part of this work, yes?

Reply:
yup, i'm talking about the part from your screenshot. roles and raids don't match, so aligning the spacings between all these pages would be really nice. but maybe we can wait for the designers to decide on the exact numbers

Reply:
I've made the spacing match - 32px at md and higher